src/pages/_app.tsx

```diff
@@ -0,0 +1,45 @@
+import * as Sentry from '@sentry/browser';
```

Comment:
What decides whether a page's name should be underscore-prefixed or not?

Reply:
Underscore-prefixed pages are built-in components of Next. Components like `_document` which renders the standard HTML markup (like an `index.html` in Create React App), and `_app` which is rendered as the main file, and wraps around all other pages (useful for including the same logic on all your pages, like a header).

These components have defaults, but we overwrite them when we create a file with the given name.